mmcv/runner/base_module.py

```diff
@@ -193,3 +193,16 @@ class ModuleList(BaseModule, nn.ModuleList):
     def __init__(self, modules=None, init_cfg=None):
         BaseModule.__init__(self, init_cfg)
         nn.ModuleList.__init__(self, modules)
+
+
+class ModuleDict(BaseModule, nn.ModuleDict):
+    """ModuleDict in openmmlab.
+
+    Args:
+        modules (dict, optional): an dict of modules to add.
```

Comment:
a mapping (dictionary) of (string: module) or an iterable of key-value pairs of type (string, module)